src/main/features/ccj/routes/their-details.ts

```diff
@@ -43,8 +43,9 @@ export default express.Router()
         if (form.hasErrors()) {
           renderView(form, res)
         } else {
+          const { externalId } = req.params
           user.ccjDraft.defendant.partyDetails = form.model
           await DraftCCJService.save(res, next)
-          res.redirect('todo')
+          res.redirect(Paths.paidAmountPage.uri.replace(':externalId', externalId))
```

Comment:
maybe leave this as todo? as this page isn't the next in the flow.
Coordinate with Ranjit to get this story merged first